index.html

```diff
@@ -81,7 +82,7 @@ <h2>LET’S CONNECT</h2>
 
     <!-- WORK SECTION START  -->
     <section id="work">
-        <section class="project-section">
+        <!-- <section class="project-section">
```

Comment:
- [ ] please avoid commenting out any unused code, this will make your project look unorganised and harder to read and is not in line with the general requirements.

Reply:
thank you very much for the clear instructions @mikethreels